indra/newview/llfilepicker.cpp

```diff
@@ -649,6 +649,9 @@ std::unique_ptr<std::vector<std::string>> LLFilePicker::navOpenFilterProc(ELoadF
         case FFLOAD_IMAGE:
             allowedv->push_back("jpg");
             allowedv->push_back("jpeg");
+            allowedv->push_back("j2c");
+            allowedv->push_back("jp2");
+            allowedv->push_back("png");
```

Comment:
duplicate "png" line

Reply:
Thank you, fixed. Will doublecheck it again before merge.